woocommerce-google-analytics-integration.php

```diff
@@ -208,6 +209,26 @@ public function maybe_show_ga_pro_notices() {
 			update_option( 'woocommerce_google_analytics_pro_notice_shown', true );
 		}
 
+		/**
+		 * Set default options during activation if no settings exist
+		 *
+		 * @since x.x.x
+		 *
+		 * @return void
+		 */
+		public function maybe_set_defaults() {
+			$settings_key = 'woocommerce_google_analytics_settings';
+
+			if ( false === get_option( $settings_key, false ) ) {
+				update_option(
+					$settings_key,
+					array(
+						'ga_product_identifier' => 'product_id',
```

Comment:
Shouldn't be this `product_sku`? As the default setting?

Reply:
I see it's intentional for new install but just wonder why we change from SKU to ID now.

Reply:
As part of the project to remove Universal Analytics we're also moving to using the product ID as the standard identifier (peeuvX-SJ-p2). One reason is to allow merchants using GLA to have the same product identification across both extensions by using the `woocommerce_ga_product_identifier` filter to prefix the ID with `gla_`.

Aside from that, the product ID is a constant and reliable way to identify a product whereas the SKU + fallback is more likely to cause issues for example if a merchant sets a SKU for an existing product or changes it then it would now be a "new" product in Google Analytics and not properly link up with historical data.